webapp/tokens/flask.py

```diff
@@ -0,0 +1,55 @@
+import flask
+
+import webapp.tokens.mapper as token_mapper
+from webapp.decorators import token_required
+
+
+tokens_blueprint = flask.Blueprint("tokens", __name__)
```

Comment:
I'm really not a fan of creating a blueprint for every subpath. To me it just seems like you're just adding complex namespacing that is gonna make it harder to find stuff. Java-style.

Couldn't we just keep all views in a `views.py` or something until we have a compelling reason to create a Blueprint?